pages/docs/reference/idioms.md

```diff
@@ -358,3 +358,13 @@ if (b == true) {
 }
 ```
 </div>
+
+### Swapping two variables
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+```kotlin
```

Comment:
add \n after div

Reply:
After which div? I've added one more to the end of file. I don't see line breaks being 
used anywhere else in this file.